main.go

```diff
@@ -17,3 +32,39 @@ func main() {
 	r.POST("/commands", views.NewQuoteHandler(slackClient, signingSecret).QuoteServe)
 	r.Run()
 }
+
+func ParseFinancialTable(url string) []Table {
+	document, err := goquery.NewDocument(url)
+	if err != nil {
+		panic(err)
+	}
+	raw_tables := document.Find("div.month table")
+	all_tables := []Table{}
+	raw_tables.Each(func(i int, s *goquery.Selection)  {
+		caption := s.Find("caption").Text()
+		table := Table{caption: caption, values: []TableEntry{}}
+	    s.Find("tbody tr").Each(func(i int, s *goquery.Selection)  {
+			name := s.Find("td .name").Text()
+			value := StringToFloat(s.Find("td .value"))
+			table.values = append(table.values, TableEntry{name: name, value: value})
+		})
+		all_tables = append(all_tables, table)
+	})
+
+	for i:=0; i<len(all_tables); i++ {
+		fmt.Println(all_tables[i].caption)
+	}
```

Comment:
Given that this is slice, wouldn't that be better?
```suggestion
	for idx, table := range all_tables {
		fmt.Println(table.caption)
	}
```